README.md

```diff
@@ -2,27 +2,37 @@
 Response management Concourse pipelines and documentation
 
 ## Pipelines
-### CI Kubernetes Pipeline
-Specified in `pipelines/ci-kubernetes-pipeline.yml`, this pipeline has three sections.
+### [CI Kubernetes Pipeline](pipelines/ci-kubernetes-pipeline.yml)
+This pipeline has three sections:
 
-The first pulls from the [`census-rm-kubernetes`](https://github.com/ONSdigital/census-rm-kubernetes) repo triggered by commits to master and applies the latest microservice and handler configs to the kubernetes cluster specified by the pipeline configuration.
+1.The first pulls from the [`census-rm-kubernetes`](https://github.com/ONSdigital/census-rm-kubernetes) repo triggered by commits to master and applies the latest microservice and handler configs to the kubernetes cluster specified by the pipeline configuration.
 
-The second checks for new builds for the `latest` tag for the RM container registry images. On being triggered by a new image build it runs the config apply to ensure the config is in line with master, then runs `kubectl patch` for the newly built images deployment giving them a timestamp and image digest label. This causes kubernetes to re-pull the latest image and bring up new pods while we are using `imagePullPolicy: Always`.
+1.The second checks for new builds for the `latest` tag for the RM container registry images. On being triggered by a new image build it runs the config apply to ensure the config is in line with master, then runs `kubectl patch` for the newly built images deployment giving them a timestamp and image digest label. This causes kubernetes to re-pull the latest image and bring up new pods while we are using `imagePullPolicy: Always`.
```

Comment:
I'm not sure if this counter is formatted properly?
![image](https://user-images.githubusercontent.com/43209528/57513778-f6418d80-7306-11e9-8a6a-1822cb06d541.png)

Reply:
Yeah, that wasn't meant to be there like that 👍